src/lib/lit-element-renderer.ts

```diff
@@ -30,24 +30,32 @@ export class LitElementRenderer implements ElementRenderer {
     const renderResult = ((instance as unknown) as {
       render(): TemplateResult;
     }).render();
-    yield '<shadow-root>';
+    yield '<template shadowroot="open">';
+    yield* this.renderStyles(instance);
     yield* render(renderResult);
-    yield '</shadow-root>';
+    yield '</template>';
   }
 
   *renderStyles(
-    _elementClass: Constructor<HTMLElement>
-  ): Iterator<string> {
-    const scopedStyles = [];
-    for (const [tagName, definition] of (customElements as any).__definitions) {
-      const styles = [(definition.ctor as any).styles].flat(Infinity);
-      for (const style of styles) {
-        if (style instanceof CSSResult) {
-          const scoped = StyleTransformer.css(style.cssText, tagName);
-          scopedStyles.push(scoped);
+    instance: LitElement
+  ): IterableIterator<string> {
+    const ctor = customElements.get(instance.tagName);
```

Comment:
I think this should just be moved into `renderElement`

Reply:
Done.